/**:
  ros__parameters:
    speed_scale: 1.0
    timer_hz: 40.0
    joy_button_index: 2
    ack_button_index: 3

## button index (dual shock 4)
# 0: ×
# 1: ○
# 2: △
# 3: □
# 4: L1
# 5: R1
# 6: L2
# 7: R2
# 8: share
# 9: option
# 10: home
# 11: L3
# 12: R3